.github/workflows/mark-issue-stale.yml

```diff
@@ -12,16 +12,23 @@ jobs:
       - uses: actions/stale@v3
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+          # Message to be added to stale issues.
           stale-issue-message: 'This issue is stale because it has been 180 days with no activity. You can keep the issue open by adding a comment. If you do, please provide additional context and explain why you’d like it to remain open. You can also close the issue yourself — if you do, please add a brief explanation and apply one of relevant issue close labels.'
           # Days before issue is considered stale.
           days-before-issue-stale: 180
-          # Exempt issue labels.
-          exempt-issue-labels: '[Pri] High,[Pri] BLOCKER,[Status] Keep Open'
+          # Exempted issue labels.
+          exempt-issue-labels: '[Pri] High,[Pri] BLOCKER,[Status] Keep Open,[Status] Blocked / Hold'
+          # Message to be added to stale PRs.
+          stale-pr-message: 'This PR has been marked as stale due to lack of activity within the last 30 days.'
+          # Exempted PR labels.
+          exempt-pr-labels: '[Pri] High,[Pri] BLOCKER'
```

Comment:
Should this list be the same as the exempt-issue-labels?

Reply:
Ah yes, good point! 

It looks like `Keep Open` label no longer exists as a label. I recall it used, so not sure where it went.

I did not include the label for `Blocked / Hold` because I __thought__ there were no PRs that had used the label, but looks like that's my own mistake in querying.